### DISCUSSION TITLE
I want audio/ogg support in Media Source Extensions

### DISCUSSION THREAD

**Discussion by aarongustafson** [WebWeWant/webwewant.fyi#166] [Wants]
Browsers can currently play `audio/ogg; codec=opus` files natively with `audio` elements, but that MIME is not consistently supported for Media Source Extensions (MSE). MSE can solve the [problem of low-latency audio playback](https://github.com/xiph/opus-tools/issues/49), and the Opus audio codec is one of best performing low-latency codecs available to us. Unfortunately, Ogg containers are the ones officially used for Opus files, and MSE does not support them. The current work-around is to use third-party tools like FFMPEG to create WebM Opus files, which does not align with the Opus file standards and current Opus tools.

<hr>

https://webwewant.fyi/wants/108/

**Comment by guest271314** [comment]
This will eliminate the need to encode as WebM or Matroska first.
